ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.sass

```diff
@@ -1,6 +1,6 @@
 .btn-group
   position: fixed
-  z-index: 1
+  z-index: 3
```

Comment:
Could you please clarify why there should be 3 here?

Reply:
fixed